Streetcode/Streetcode.WebApi/Controllers/Streetcode/TextContent/TextController.cs

```diff
@@ -28,9 +29,9 @@ public async Task<IActionResult> GetByStreetcodeId([FromRoute] int streetcodeId)
         return HandleResult(await Mediator.Send(new GetTextByStreetcodeIdQuery(streetcodeId)));
     }
 
-    [HttpGet]
-    public async Task<IActionResult> GetParsedText([FromQuery] string text)
+    [HttpPost]
+    public async Task<IActionResult> GetParsedText([FromBody] TextPreviewDTO text)
     {
-        return HandleResult(await Mediator.Send(new GetParsedTextForAdminPreviewCommand(text)));
+        return HandleResult(await Mediator.Send(new GetParsedTextForAdminPreviewCommand(text.TextContent)));
```

Comment:
Maybe we can update these Get prefixes with something like "Update..." to fit more for Post method